DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -273,10 +335,16 @@ static function bool BingoGoalImpossible(string bingo_event, int start_map)
         case "MetSmuggler":
             return start_map>=80; //Mission 8 and later starts you should already know Smuggler (see StartMapSpecificFlags)
         case "KnowsGuntherKillphrase":
+            if (end_mission < 12){
+                return True;
+            }
             return start_map>=60; //Have to have told Jaime to meet you in Paris in mission 5 to get Gunther's killphrase
```

Comment:
should we default to Jaime being in Paris if you start after mission 5?

Reply:
We *could* also default to him being in Hong Kong if you start in mission 6, and Paris if you start later than that.  I'm not opposed to either.